rdmo/accounts/socialaccount.py

```diff
@@ -0,0 +1,45 @@
+from django.conf import settings
```

Comment:
I needed to refactor this to keep the use of `SOCIALACCOUNT` related imports  separate from the normal account imports

Reply:
Maybe `adapter/account.py` (with imports in `adapter/__init__.py` to keep the existing settings working) and `adapter/socialaccount.py`? Or `adapter_social.py`? In the end its analog to the `adapter.py` file, right? We could also keep `socialaccount.py` if you don't like the suggestions.

Reply:
yes, seems also fine to me. Problem for the name 'adapter' is that there is not only an adapter but also a form in the current `adapter.py` ;)

Reply:
Yes, we could call it `account.py` and `socialaccount.py` and change `settings.py`. I think noone used the filename in their settings.

Reply:
ok, Ive changed it to `account.py`. It can be refactored into a folder when it's needed..